cogs/modmail.py

```diff
@@ -2186,6 +2187,56 @@ async def isenable(self, ctx):
 
         return await ctx.send(embed=embed)
 
+    @commands.command(name="export")
+    @checks.has_permissions(PermissionLevel.ADMINISTRATOR)
+    async def export_backup(self, ctx, collection_name):
+        """
+        Export a backup of a collection in the form of a json file.
+
+        {prefix}export <collection_name>
```

Comment:
The two usage examples, on L2196 and L2210, are unnecessary, since discord.py auto generates the usage spec. However, it might be useful to note in the import command's help doc that you should attach the backup file to the message.